src/idsk/components/stepper/stepper.scss

```diff
@@ -210,6 +210,19 @@
     margin-bottom: 0;
   }
 
+  @include mq ($from: mobile) {
+    .idsk-stepper__circle{
+      width: 26px;
+      height: 26px;
+    }
+  }
+
+  @include mq ($from: mobile) {
+    .idsk-stepper__circle--number{
+      line-height: 0.9rem;
+    }
+  }
+
```

Comment:
v kode je uz implementacia tychto elementov, trebalo by to presunut na jedno miesto